### ISSUE TITLE
GFM: How to include code for a fenced code block inside a fenced code block?

### ISSUE THREAD

**Issue by LeaVerou** [showdownjs/showdown#456]
The [GFM spec](https://github.github.com/gfm/#fenced-code-blocks) includes multiple strategies for this, none of which seem to work with Showdown.

- "The closing code fence must use the same character as the opening fence:": Tildes don't seem to be supported
- "The closing code fence must be at least as long as the opening fence": Using more than three backticks doesn't seem to be supported
- I tried backslash escaping, that didn't work either.

Is there any way to do this?

(code below for attempt number 2: using different number of backticks)

~~~markup
## Demo

````markup
<div mv-app mv-storage="local" 
     mv-plugins="markdown">
	<div property="text" class="markdown">### Heading
**This is bold** *This is italic* 
Here’s a [link!](https://mavo.io)

And some code:

```js
	foo.bar();
```
	</div>
</div>
````
~~~

**Comment by tivie** [comment]
Tilde fenced blocks is, indeed, a feature implemented (not so) recently in github that is missing from showdown. **I will implement it in the next showdown release.**

Currently there is no way to include *code for a fenced code block inside a **fenced** code block*.

But meanwhile,  you can include *code for a fenced code block inside an **indented** code block**, like so:

~~~
    ```markup
    <div mv-app mv-storage="local" mv-plugins="markdown">
    <div property="text" class="markdown">### Heading
    **This is bold** *This is italic* 
    Here’s a [link!](https://mavo.io)

    And some code:

        ```js
        foo.bar();
        ```
	</div>
    </div>
    ```
~~~

**Comment by LeaVerou** [comment]
But there's no way to provide a language tag for an indented code block, right?

**Comment by tivie** [comment]
@LeaVerou right

**Comment by tivie** [comment]
Should be fixed now (c956ede)

---

>As you know, ShowdownJS is a free library and it will remain free forever. However, maintaining and improving the library costs time and money. Currently, we're looking to improve showdown with automated tests in all browsers and a proper domain and webpage. 500$ should be enough to to keep showdown testing framework running for a year or two.

**If you like our work and find our library useful, please donate [through Pledgie](https://pledgie.com/campaigns/34908) or directly [through paypal](https://www.paypal.me/tiviesantos)!! Your contribution will be greatly appreciated.**
